tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -37,32 +37,31 @@ Feature: State of the quota
       | Quota99%  | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789  | critical | 100   | 1         | 99   |
       | Quota100% | 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 1234567890 | exceeded | 100   | 0         | 100  |
 
-  
-  Scenario: uploading a file with an insufficient quota
-    Given user "Alice" has created a space "Project Alfa" of type "project" with quota "10" 
+
+  Scenario: A file cannot be uploaded if there is insufficient quota
+    Given user "Alice" has created a space "Project Alfa" of type "project" with quota "10"
     When user "Alice" uploads a file inside space "Project Alfa" with content "More than 10 bytes" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "507"
 
 
-  Scenario: creating a folder with an insufficient quota
+  Scenario: A folder cannot be uploaded if there is insufficient quota
```

Comment:
```suggestion
  Scenario: A folder cannot be created if there is insufficient quota
```

Reply:
```suggestion
  Scenario: A folder can be created even if there is insufficient quota for file content
```

Reply:
@individual-it thanks for spotting. I looked more closely and a folder can be created in this case. Folders do not consume quota. So I changed even more words.